ssl/s3_lib.c

```diff
@@ -3564,16 +3565,11 @@ long ssl3_ctrl(SSL *s, int cmd, long larg, void *parg)
         break;
 
     case SSL_CTRL_GET_TLSEXT_STATUS_REQ_OCSP_RESP:
```

Comment:
This case is the code behind the method `SSL_get_tlsext_status_ocsp_resp()` , as it is defined by a macro call to `ssl3_ctrl(SSL_CTRL_GET_TLSEXT_STATUS_REQ_OCSP_RESP, ...`

Reply:
you must not change these two `case`

Reply:
It does not seem to be pushed/committed?